frontend/src/pages/CoursesList/styled.ts

```diff
@@ -42,13 +37,28 @@ export const CourseButton = styled(Button)({
 });
 
 export const CourseActions = styled('div')({
-  [theme.breakpoints.up('xs')]: {
-    display: 'block',
+  [theme.breakpoints.down('sm')]: {
+    display: 'flex',
+    flexDirection: 'column',
   },
   [theme.breakpoints.up('lg')]: {
     display: 'flex',
     justifyContent: 'space-between',
     margin: '0px',
     minWidth: '250px',
   },
+  '@media(max-width: 1230px)': {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  [theme.breakpoints.down('lg')]: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: '0px',
+    minWidth: '250px',
+  },
+  '@media(max-width: 575px)': {
+    display: 'flex',
+    justifyContent: 'center',
+  },
```

Comment:
Why have you used @media? We can control everything with breakpoints.

Reply:
Yes, I have problem with this. Because i cannot make it work `flex-wrap` -  elements want to go out of mother-box. For this i rewrote flex-wrap to flex0direction.

Reply:
Resolved this problem. Removed **@media**